tests/metrics_publishing.rs

```diff
@@ -121,15 +101,11 @@ async fn f64_up_down_counter_is_exported() {
 
 #[tokio::test]
 async fn u64_histogram_is_exported() {
-    let (subscriber, exporter) = init_subscriber(
-        "abcdefg".to_string(),
-        InstrumentKind::Histogram,
-        NumberKind::U64,
-        Number::from(9_u64),
-    );
+    let (subscriber, exporter) =
+        init_subscriber("abcdefg".to_string(), InstrumentKind::Histogram, 9_u64);
 
     tracing::subscriber::with_default(subscriber, || {
-        tracing::info!(value.abcdefg = 9_u64);
+        tracing::info!(histogram.abcdefg = 9_u64);
```

Comment:
previous histogram test span attribute prefix does not match impl (`histogram`)
https://github.com/tokio-rs/tracing-opentelemetry/blob/7b77d33d58eea0a7aa922bb45cfbab7272559c52/src/metrics.rs#L17